### DISCUSSION TITLE
Memory Mapping / RAM Addressing of an ECU

### DISCUSSION THREAD

**Discussion by cqke** [cedricp/ddt4all#1211] [Q&A]
Hello,
Thanks for your tool, which is awesome.
I'm currently looking for memory mapping or ram addressing of the x98 Clio 4 RS ECU, EMS3150 TC1767.
I'm in the process of RE the firmware and if your tool can help me, it could be awesome.
Any tips welcome !
Thanks

**Comment by Furtif** [comment]
?! 🤔

**Reply by cqke** [reply]
Yes ?

**Comment by Marianpol** [comment]
DDT4ALL is almost useless in your case. Get A2L file for your ECU and load it into IDA Pro/Ghidra.

**Reply by cqke** [reply]
Getting A2L for an ECU is almost impossible. Of course it could help me very much ^^.
In fact, i'm looking for all things which can ease the analysis of the code of the firmware (which I successfully disassembled)

**Comment by EngineerTunerNorway** [comment]
I see the thread is a bit old. But i have mappack for Renlaut

Renaut Clio IV RS 200 Siemens EMS3150 237103333R RDB2160300000 - (.ols)
Renault\EMS3150 - (.rar)

**Reply by johnnebraskared-create** [reply]
Are the files available?
